docs/user_manual.md

```diff
@@ -878,6 +879,157 @@ datum stats -p test_project
 
 </details>
 
+
+### Validate project annotations
+
+This command inspects annotations based on the task type
+and stores the result in JSON file.
+
+The task types supported are `classification`, `detection`, and `segmentation`.
+
+The validation result contains
+- annotation statistics based on the task type
+- validation reports, such as
+    - items not having annotations
+    - items having undefined annotations
+    - imbalanced distribution in class/attributes
+    - too small or large values (items_far_from_mean)
+- summary
+
+Usage:
+
+``` bash
+datum validate --help
+
+datum validate -p <project dir> <task_type>
+```
+
+Validation Result:
+
+<details>
+
+``` bash
+{
+    'statistics': {
+        ## common statistics
+        'label_distribution': {
+            'defined_labels': <dict>,   # <label:str>: <count:int>
+            'undefined_labels': <dict>
+            #  <label:str>: {
+            #    'count': <int>,
+            #    'items_with_undefined_label': [(<item.id>, <item.subset>), ]
```

Comment:
Please be consistent in field definitions. 
It is unclear, what the dot means and what type the fields have.

Reply:
`item` means  `DatasetItem`. I'll correct this.